stylesheets/grid/modern/_grid.styl

```diff
@@ -98,6 +98,9 @@
       margin-top: $sizing-400
 
   @media $media-xlarge
+    &--3
```

Comment:
That's not consistent in this file. See `--6`, `--7`, `--8`, `--9`, `--10`, `--11`, and `--12` above.